### PR TITLE
Zhiliu/load themable style remove dead code

### PR DESCRIPTION
**Pull request by jijizz**
remove dead code in load-themable-style that for IE9, but we have already stopped support for IE9

### CODE REVIEW

**Comment by nickpape-msft** [comment]
@iclanton Can you take a look at this?

File: libraries/load-themed-styles/src/index.ts [hunk 1]
```diff
@@ -110,9 +106,6 @@ const _themeState: IThemeState = initializeThemeState();
 // tslint:disable-next-line:max-line-length
 const _themeTokenRegex: RegExp = /[\'\"]\[theme:\s*(\w+)\s*(?:\,\s*default:\s*([\\"\']?[\.\,\(\)\#\-\s\w]*[\.\,\(\)\#\-\w][\"\']?))?\s*\][\'\"]/g;
 
-/** Maximum style text length, for supporting IE style restrictions. */
```
Comment (iclanton) [inline libraries/load-themed-styles/src/index.ts]: Is this possibly why we've seen regressions in IE lately?

File: libraries/load-themed-styles/src/index.ts [hunk 2]
```diff
@@ -409,64 +397,4 @@ function registerStyles(styleArray: ThemableArray): void {
   } else {
     _themeState.registeredStyles.push(record);
   }
-}
-
-/**
- * Registers a set of style text, for IE 9 and below, which has a ~30 style element limit so we need
- * to register slightly differently.
```
Comment (iclanton) [inline libraries/load-themed-styles/src/index.ts]: Is the fact that this code is now "dead" potentially a reason for #388?

**Comment by pgonzal** [comment]
Closing due to a long period of inactivity. If you still want to implement this, please reopen the PR. Thanks!

**Pull request closed without merging**